doctrine-mapping.xsd

```diff
@@ -202,7 +202,7 @@
 
   <xs:simpleType name="tablename" id="tablename">
     <xs:restriction base="xs:token">
-      <xs:pattern value="[a-zA-Z_u01-uff.]+" id="tablename.pattern">
+      <xs:pattern value="([&#x20;-&#x7F;-[\.]]+\.)?([a-zA-Z][a-zA-Z0-9_]*|`[&#x20;-&#x7F;]+`)" id="tablename.pattern">
```

Comment:
Note: the backslash before the dot in the subtracted character class is "redundant", but without it PhpStorm flags a syntax error "Unclosed character class"

Reply:
Wait, wouldn't ``"([a-zA-Z][a-zA-Z0-9_]*\.)?[a-zA-Z][a-zA-Z0-9_]*|([&#x20;-&#x7F;-[`\.]][&#x20;-&#x7F;-[\.]]*\.)?`[&#x20;-&#x7F;]+`"`` be better? (the schema part cannot be backquoted, and will be SQL-quoted only if the table part is backquoted)

Reply:
I have opened a separate #7329 about the peculiar schema part quoting behavior. But for now this PR should be fine as it is.